appengine-push/README.md

```diff
@@ -55,7 +55,7 @@ $ appcfg.py --oauth2 update -A your-application-id .
 or you can use gcloud preview feature
 
 ```
-$ gcloud preview app deploy --project your-application-id .
+$ gcloud app deploy --project your-application-id .
```

Comment:
You don't need to specify the project or directory anymore, just `gcloud app deploy`